src/api/app/assets/stylesheets/webui2/build-results.scss

```diff
@@ -23,31 +23,46 @@
 }
 
 .build-state-building {
-  color: $blue;
+  @extend .text-white;
+  @extend .px-1;
+  background-color: $blue;
 }
 
 .build-state-scheduled {
-  color: $cyan;
+  @extend .text-white;
+  @extend .px-1;
+  background-color: $cyan;
 }
 
 .build-state-signing, .build-state-finished {
-  color: $dark;
+  @extend .text-white;
+  @extend .px-1;
+  background-color: $dark;
```

Comment:
I think this should be yellow somehow. Both are intermediate steps between building and the final result (green/red). You can not really do much about this state, it shouldn't be as heavy as diabled/blocked.

Reply:
@hennevogel done 
![Screenshot-2019-8-26 Open Build Service](https://user-images.githubusercontent.com/22001671/63685642-2d797300-c800-11e9-8f74-15e0fcc7d5f6.png)